src/LearnositySdk/Request/Init.php

```diff
@@ -17,6 +18,12 @@
 
 class Init
 {
+    /**
+     * We use telemetry to gather data about platforms people run our SDKs on. By default it's
```

Comment:
... to enable better support and feature planning.

Reply:
Ha! you a salesman? ;) sounds better indeed, cheers

Reply:
Ha! Hopefully not. But I know how I react to telemetry: unless it has some ultimate benefit for me, I don't want it.